webpack.config.js

```diff
@@ -4,10 +4,67 @@ const webpack           = require('webpack')
 
 const packageInfo       = require('./package')
 
+const bodyParser        = require('webpack-body-parser')
+
 process.env.NODE_ENV = process.env.NODE_ENV || 'production'
 
 const isDev = process.env.NODE_ENV === 'development'
 
+const CHUNK_SIZE = 1048576
+const ChunkActiveUploads = {}
+
+const chunkUploadStart = (req, res) => {
+  const uuid = Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1)
+  ChunkActiveUploads[uuid] = {}
+
+  return res.json({
+    status: 'success',
+    data: {
+      session_id: uuid,
+      start_offset: 0,
+      end_offset: CHUNK_SIZE
+    }
+  })
+}
+
+const chunkUploadPart = (req, res) => {
+  setTimeout(() => {
+    const rand = Math.random()
+    if (rand <= 0.25) {
+      res.status(500)
+      res.json({ status: 'error', error: 'server' })
+    } else {
+      res.send({ status: 'success' })
+    }
+  }, 100 + parseInt(Math.random() * 2000, 10))
+}
+
+const chunkUploadFinish = (req, res) => {
```

Comment:
Same as the `chunkUploadPart` function. Maybe just use the same function or let the one call the other, since it's for testing purpose.